jquery.mosaic.js

```diff
@@ -41,6 +41,9 @@
 
 			if (o.refitOnResize)
 				$(window).on('resize', null, null, function() {
+				  if (base.$el.is(':hidden')) {
```

Comment:
I think that's a nice idea, I will implement implement this same solution but using `$(base.el).is(':hidden')` instead to keep it coherent with the way the element is addressed throughout the rest of the code. Thank you!